pyproject.toml

```diff
@@ -100,6 +100,7 @@ omit = [
     "*/apps.py",
     "*/admin.py",
     "manage.py",
+    "timed/redmine/management/commands/import_project_data.py",
```

Comment:
baye a `# TODO` might make sense here

(yes i'm aware that more `todo:` culture would also be required for this to help....)

let me think some and i'll turn this into a new issue and resolve it here

Reply:
yea, AFAIK that file wasn't picked up by pytest previously

Reply:
just noticed that it's in `timed/redmine` let's consider it to be obsolecent soon and then this is fine as is